### PR TITLE
Yotam playg 37 support insert fix

### PR DESCRIPTION
**Pull request by yotam-yakovw**
New and improved branch for my ticket
Commits went down to 2 and only relevant files are commited

### CODE REVIEW

File: main_flow/main_flow.py
Comment (smartFreshBear) [inline main_flow/main_flow.py:109]: why did you change the stemmed to steamed? you added a typo upon us!

**Review by smartFreshBear — CHANGES_REQUESTED** [review]
last small comment to fix and I will approve (squash the this small fix to the main one!!)

**Comment by yotam-yakovw** [comment]
GIT IS HARD!
so I fixed the typo and commited the changes, then squashed the commit to the previous so it won't show but to push I was forced to merge the remote branch(?)
Now there is a copy of the last commit and another merge commit? what is going on and what am I doing wrong??

**Comment by smartFreshBear** [comment]
yes because!!!
you merged instead rebase right?  like I said many times in Playground we don't merge...
but no worries.
you can keep the merge commit. 
but the rest you have to squash together

**Comment by yotam-yakovw** [comment]
But it didn't let me push without merging I'm so confused
Was I supposed to pull and then push? Wouldn't it change things back again?

**Comment by yotam-yakovw** [comment]
So I looked it up online, apparently because the commits were already in the remote branch I need to force push to change commits and that let me get pass the merge requirement.
somehow the merge commit disappeared so that's a plus I guess :D

**Review by smartFreshBear — APPROVED** [review]
approved, please verify u can train the model & send request successfully before "rebase & merge"

**Comment by yotam-yakovw** [comment]
After logging branch revert rebased on master, fixed conflicts and tested the app
Force pushed changes and hopefully ready to merge

**Review by smartFreshBear — APPROVED** [review]

**Pull request merged**
